src/Components/CreateImageWizard/CreateImageWizard.tsx

```diff
@@ -97,41 +104,70 @@ export const CustomWizardFooter = ({
   beforeNext,
   optional: optional,
 }: CustomWizardFooterPropType) => {
-  const { goToNextStep, goToPrevStep, goToStepById, close } =
+  const { goToNextStep, goToPrevStep, goToStepById, close, activeStep } =
     useWizardContext();
+  const { analytics } = useChrome();
+  const nextBtnID = 'wizard-next-btn';
+  const backBtnID = 'wizard-back-btn';
+  const reviewAndFinishBtnID = 'wizard-review-and-finish-btn';
+  const cancelBtnID = 'wizard-cancel-btn';
   return (
     <WizardFooterWrapper>
       <Button
-        ouiaId="wizard-next-btn"
+        ouiaId={nextBtnID}
         variant="primary"
         onClick={() => {
+          analytics.track(`${AMPLITUDE_MODULE_NAME} - Button Clicked`, {
+            button_id: nextBtnID,
+            active_step_id: activeStep.id,
+          });
```

Comment:
Looks good to me! Could you please add a name of the module to the tracked data of all events as well? It's a pattern we've started with the recommendations, from what I understood it should make working with the data a tad bit easier.

```suggestion
          analytics.track(`${AMPLITUDE_MODULE_NAME} - Button Clicked`, {
            module: AMPLITUDE_MODULE_NAME,
            button_id: nextBtnID,
            active_step_id: activeStep.id,
          });
```

Reply:
Sure! I added it :sparkles: